### PR TITLE
Feature/refactor

### PR DESCRIPTION
**Pull request by surabhil**

### CODE REVIEW

File: server/lib/server.js [hunk 1]
```diff
@@ -7,26 +7,48 @@ const {
   logError
 } = require('./logging')
 
-function checkRequestParams(reqParamHash, requestParams) {
+// append url/body to the parameter name to be able to distinguish b/w url and body parameters
+async function getParams(requestParams) {
```
Comment (blackwatertepes) [inline server/lib/server.js]: Doesn't need to be async?

File: client/src/client.js [hunk 2]
```diff
@@ -72,67 +87,92 @@ class ApiMarketClient {
...
       const errMsg = `Problem retrieving info from the ORE blockchain. Config file expects an ORE node running here: ${this.oreNetworkUri}. ${error}`
       reject(error)
     }
 
     //console.log(`oreNetworkUri ${this.oreNetworkUri}   chain_id ${this.OreChainId}`)
   }
 
-  async getOptions(endpoint, httpMethod, oreAccessToken, requestParams){
+  // append url/body to the parameter name to be able to distinguish b/w url and body parameters
+  async getParams(requestParams) {
```
Comment (blackwatertepes) [inline client/src/client.js]: This looks the same as the previous getParams method. Can we DRY this up, by refactoring into a shared helper?

Comment (surabhil) [reply client/src/client.js]: the client and server library are separate and installed separately. I don't think we can have a shared helper.

File: server/lib/server.js (hunk 1 above)
Comment (surabhil) [reply server/lib/server.js]: Agreed! Removed async.

**Pull request merged**